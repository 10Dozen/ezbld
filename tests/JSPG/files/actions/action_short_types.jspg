# Scene

@ Action1 | <

@ Action2 | >

@ Action3 | <# pic.jpg

@ Action4 | #> pic.jpg

@ Action5 | T

@ Action6 | ST

@ Action7 | C

@ Action8 | HID

@ Action9 | > | MyTag

@ Action10 | <# pic.jpg | MyTag
