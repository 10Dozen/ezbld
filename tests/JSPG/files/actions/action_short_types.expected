[0]
cls = JSPGScene
name = Scene | TEXT

[1]
cls = JSPGAction
name = Action1 | TEXT
type = scene_left | TEXT
scene = Scene | TEXT

[2]
cls = JSPGAction
name = Action2 | TEXT
type = scene_right | TEXT
scene = Scene | TEXT

[3]
cls = JSPGAction
name = Action3 | TEXT
type = dialog_left | TEXT
portrait = pic.jpg | TEXT
scene = Scene | TEXT

[4]
cls = JSPGAction
name = Action4 | TEXT
type = dialog_right | TEXT
portrait = pic.jpg | TEXT
scene = Scene | TEXT

[5]
cls = JSPGAction
name = Action5 | TEXT
type = title | TEXT
scene = Scene | TEXT

[6]
cls = JSPGAction
name = Action6 | TEXT
type = subtitle | TEXT
scene = Scene | TEXT

[7]
cls = JSPGAction
name = Action7 | TEXT
type = container | TEXT
scene = Scene | TEXT

[8]
cls = JSPGAction
name = Action8 | TEXT
type = hidden | TEXT
scene = Scene | TEXT

[9]
cls = JSPGAction
name = Action9 | TEXT
type = scene_right | TEXT
tag = MyTag | TEXT
scene = Scene | TEXT

[10]
cls = JSPGAction
name = Action10 | TEXT
type = dialog_left | TEXT
portrait = pic.jpg | TEXT
tag = MyTag | TEXT
scene = Scene | TEXT
